### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by rgralakosi**
Update edgecmd's release notes.

### CODE REVIEW

File: content/release-notes/release-notes.md [hunk 1]
```diff
@@ -16,13 +16,23 @@ For more information, see [EdgeCmd utility](xref:index).
...
 ### Enhancements
 
 Changes from version 1.1
 
-* Support of the data source discovery feature
+* Added support for datasource discovery.
+* Added *cancel* and *resume* keywords for historyrecoveries and *cancel* for discoveries.
+* Added support for CSV import and export of complex types. 
+* Improved help messages.
+* Built Edgecmd as a .NET 5 console application, which reduces its size and improves startup performance.
```
Comment (zryska) [inline content/release-notes/release-notes.md:31]: The runtime has been upgraded to .NET 5.0 (might be better)

File: content/release-notes/release-notes.md [hunk 2]
```diff
@@ -16,13 +16,23 @@ For more information, see [EdgeCmd utility](xref:index).
...
+* Enhanced the Windows installer:
```
Comment (zryska) [inline content/release-notes/release-notes.md:32]: I would say only 
- Enhanced Windows installation experience
- EdgeCmd is now added to the PATH on Windows systems

File: content/release-notes/release-notes.md [hunk 3]
```diff
@@ -16,13 +16,23 @@ For more information, see [EdgeCmd utility](xref:index).
 
 ### Fixes
 
-The port configuration has been reworked, so that two ports can be configured. One for communication and one for the payload.
+* The port configuration has been reworked, so that two ports can be configured. One for communication and one for the payload.
```
Comment (zryska) [inline content/release-notes/release-notes.md:19]: we might replace reworked with enhanced

**Review by zryska — APPROVED** [review]

File: content/release-notes/release-notes.md [hunk 4]
```diff
@@ -16,13 +16,23 @@ For more information, see [EdgeCmd utility](xref:index).
...
+* Extra arguments in a command line with the *help* keyword are detected and an error message returned.
+* Reworded and fixed an error in the edgecmd message presented when trying to add a component using an input file.
```
Comment (zryska) [inline content/release-notes/release-notes.md:21]: Fixed an error might be enough here.

Comment (rgralakosi) [reply content/release-notes/release-notes.md:21]: Done! Thanks!

**Comment by TinaUll** [comment]
@rgralakosi and @zryska Let me know if I can commit my suggested changes. Thanks!

**Review by zryska — APPROVED** [review]

**Review by Xiaoli-OSI — APPROVED** [review]

**Review by TinaUll — APPROVED** [review]

**Pull request merged**